app/core/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.conf import settings
+
+
+class Recipe(models.Model):
+    """A Recipe object"""
+    name = models.CharField(max_length=255, blank=False, default=None)
```

Comment:
Because we always use Postgres as our DB engine [unless you want to specify a limit, using `TextField` is best (since it doesn't require `max_length`).](https://stackoverflow.com/a/7354680)